app/core/models.py

```diff
@@ -153,8 +153,10 @@ def party_short(self):
 
     @property
     def state_name(self):
-        if self.state:
-            return self.state.name
+        if self.state.count() > 1:
+            return '%s %s' % (self.state.count(), _('states'))
```

Comment:
Could you do state.count() only once? Otherwise this will do the count query twice.. Also, it'd be a good idea to put the placeholder into the translation text, as in some languages, the order might be reversed.. So - something like this:

```python
state_count = self.state.count()
# ...
if state_count > 1:
    return (_('%d states') % state_count)
# ...
```